recipe/meta.yaml

```diff
@@ -31,6 +31,32 @@ requirements:
     - crosstool-ng >=1.24.0,<1.25
 
 outputs:
+  - name: libgcc-devel_{{ ctng_target_platform }}
+    script: install-libgcc-devel.sh
+    build:
+      number: {{ build_num }}
+      detect_binary_files_with_prefix: False
+      binary_relocation: False
+      missing_dso_whitelist:
+        - "*"
+    about:
+      summary: The GNU C development libraries and object files
+      home: https://gcc.gnu.org/
+      license: GPL-3.0-with-GCC-exception
+
+  - name: libstdcxx-devel_{{ ctng_target_platform }}
+    script: install-libstdc++-devel.sh
+    build:
+      number: {{ build_num }}
+      detect_binary_files_with_prefix: False
+      binary_relocation: False
```

Comment:
Again why is all binary relocation off here?

Reply:
Same as before.